project/src/main/java/com/google/sps/store/SearchStore.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps.store;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.ListMultimap;
+import com.google.sps.data.EventResult;
+import com.google.sps.data.Keyword;
+import com.google.sps.utilities.KeywordHelper;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/** Store class that uses in-memory map to hold search results. */
+public class SearchStore {
+  private ListMultimap<String, EventResult> keywordToEventResults = ArrayListMultimap.create();
+  private static final float WEIGHT_IN_TITLE = 0.7f;
```

Comment:
I think a comment to clarify where these weights came from/why one matters more would be helpful

Reply:
Done - added javadoc to helper method and clarity to comment above function.